src/libraries/Microsoft.Extensions.Configuration.CommandLine/src/PACKAGE.md

```diff
@@ -1,29 +1,44 @@
 ## About
 
+<!-- A description of the package and where one can find more documentation -->
+
 Command line configuration provider implementation for [Microsoft.Extensions.Configuration](https://www.nuget.org/packages/Microsoft.Extensions.Configuration/). This package enables you to read configuration parameters from the command line arguments of your application. You can use [CommandLineConfigurationExtensions.AddCommandLine](https://learn.microsoft.com/dotnet/api/microsoft.extensions.configuration.commandlineconfigurationextensions.addcommandline) extension method on `IConfigurationBuilder` to add the command line configuration provider to the configuration builder.
 
-For more information, see the documentation: [Command-line configuration provider](https://learn.microsoft.com/dotnet/core/extensions/configuration-providers#command-line-configuration-provider).
-
-## Example
```

Comment:
Looks like this one might need to be undone - it's removing some content and template is not filled out completely.